source/config/profileUpgradeSteps.py

```diff
@@ -71,3 +71,12 @@ def upgradeConfigFrom_3_to_4(profile):
 	except KeyError:
 		# Setting does not exist, no need for upgrade of this setting
 		log.debug("reportFontAttributes not present, no action taken.")
+
+
+def upgradeConfigFrom_4_to_5(profile):
+	""" reporting details has become enabled by default.
+	Discard aria-details setting, ensure users are aware of the setting.
+	The setting was used while the feature was in development.
+	Prevented reporting 'has details' with no way to report the details.
+	"""
+	del profile['annotations']['reportDetails']

```

Comment:
This is not safe. Please wrap this in the `try ... except KeyError`block and add additional logging similarly to what is being done for the earlier upgrade steps.

Reply:
Yep, already caught this in my prior comment. Interestingly this didn't fail for me locally. I'd like to understand why.